class.osc.edu/apps/bc_osc_rstudio_server/submit.yml.erb

```diff
@@ -1,10 +0,0 @@
----
-batch_connect:
-  template: "basic"
-script:
-  accounting_id: "<%= account %>"
-  native:
-    - "--nodes"
-    - "1"
-    - "--ntasks-per-node"
-    - "<%= num_cores %>"
```

Comment:
See https://github.com/OSC/ood_core/issues/299

Not clear why we had this in the first place, but it doesn't play nice with kubernetes, so we've just got to rely on setting the right values in the forms and letting the real `submit.yml.erb` do it's thing.